### PR TITLE
Added show and shows

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Feature

This PR resolves [CX-1815]

### Description

Added show and shows in OwnerType. 

### PR Checklist (tick all before merging)

- [x] If I've added a new file to the tree I've exported it from the common `index.ts`
- [x] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[CX-1815]: https://artsyproduct.atlassian.net/browse/CX-1815?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — CHANGES_REQUESTED** [review]
Hmm, the type check fails 🤔

**Review by olerichter00 — APPROVED** [review]
Looks good to me 🛤️

File: src/DeprecatedHelpers/Tap/__tests__/TappedEntityGroup.test.ts [hunk 1]
```diff
@@ -128,4 +128,12 @@ describe("tappedEntityGroup", () => {
     })
     expect(event.action).toEqual("tappedFairGroup")
   })
+
+  it("Constructs TappedShowGroup events", () => {
```
Comment (pvinis) [inline src/DeprecatedHelpers/Tap/__tests__/TappedEntityGroup.test.ts:132]: same.

**Review by pvinis — CHANGES_REQUESTED** [review]
Reverting the two files in the helpers folder would make this PR ready to go. I commented on one of them with the reasons.

Also, FYI, normally someone from the data team should be changing things in the schema folder. Totally cool if you or another engineer change things in there, but make sure to talk to a data person first, and add them as a reviewer too, maybe even as an assignee.

**Pull request merged**